doc/NextGen_ON_CONUS.md

```diff
@@ -0,0 +1,173 @@
+
+# Build the Realization Configurations
+
+The realization configuration file in `Json` format contains high level information to run a `ngen` simulation, such as interconnected submodules, paths to forcing file, shared libraries, initialization parameters, duration of simulation, I/O variables, etc. We have built the realization configurations for several commonly used submodules which are located in `data/baseline/`. These are built by adding one submodule at a time, test run for 10 days simulation time. The successive submodules used are:
+```
+sloth (conus_bmi_multi_realization_config_w_sloth.json)
+sloth+noah-owp-modular (conus_bmi_multi_realization_config_w_sloth_noah.json)
+sloth+noah-owp-modular+pet (conus_bmi_multi_realization_config_w_sloth_noah_pet.json)
+sloth+noah-owp-modular+pet+cfe (conus_bmi_multi_realization_config_w_sloth_noah_pet_cfe.json)
+sloth+noah-owp-modular+pet+smp (conus_bmi_multi_realization_config_w_sloth_noah_pet_smp.json)
+sloth+noah-owp-modular+pet+smp+sft (conus_bmi_multi_realization_config_w_sloth_noah_pet_smp_sft.json)
+sloth+noah-owp-modular+pet+smp+sft+cfe (conus_bmi_multi_realization_config_w_sloth_noah_pet_smp_sft_cfe.json)
+```
+
+# Run Computations with Submodules
+
+With all preparation steps completed, we are now ready to run computations. We use MPI as our parallel processing application with 32 cores as an example. Users are free to choose whatever number cores they want, just make sure you will need to have the appropriate corresponding partition json file for the number of cores used. The command line for running a MPI job is as sollows:
+
+For a simple example run and quick turn around, you can run:
+
+```
+run -n 32 ./cmake_build_mpi/ngen ./hydrofabric/conus.gpkg '' ./hydrofabric/conus.gpkg '' data/baseline/conus_bmi_multi_realization_config_w_sloth.json conus_partition_32.json
+```
+
+For a more substantial example simulation, you can run:
+
+```
+run -n 32 ./cmake_build_mpi/ngen ./hydrofabric/conus.gpkg '' ./hydrofabric/conus.gpkg '' data/baseline/conus_bmi_multi_realization_config_w_sloth_noah.json conus_partition_32.json
+```
+
+For an example taken into account more realistic contributions, you can try:
+```
+run -n 32 ./cmake_build_mpi/ngen ./hydrofabric/conus.gpkg '' ./hydrofabric/conus.gpkg '' data/baseline/conus_bmi_multi_realization_config_w_sloth_noah_pet_smp_sft_cfe.json conus_partition_32.json
+```
+
+where `ngen` is the executable we build in the [Building the Executable](#Building the Executable) section. All other terms have been discussed above in details. With the current existing realization config files, the above jobs run 10 days simulation time on CONUS scale.
+
+Be aware that the above commands will generate over a million output files associated with catchment and nexus ids so if you issue a `ls` command in `ngen` directory, it will be significantly slower than usual to list all the file names. The exact time will depend on the computer you are working on.
```

Comment:
You may want to note/include the use of the `output_root` realization config option here, so that outputs can be stored in a separate directory from the source tree.